### PR TITLE
stats: Memcap pressure max relocation

### PR DESCRIPTION
**Pull request by jlucovsky**
This commit moves the memcap pressure/pressure_max stats from the global stats namespace into the memcap namespace.

With per-thread stats, they will be within the flow-manager's values.

Issue: 6398

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: [6398](https://redmine.openinfosecfoundation.org/issues/6398)

Describe changes:
- Move memcap values into their own section
- Update schema
- Formatting: `clang-format` updated several unrelated parts of the json schema

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the
pull request number.

Alternatively, `SV_BRANCH` may also be a link to an
OISF/suricata-verify pull-request.

```
SV_REPO=
SV_BRANCH=pr/1727
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPR1_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPR1_stats_chk | | |
| .uptime | 639 | 678 | 106.1% | 

Pipeline 19772

File: etc/schema.json [hunk 1]
```diff
@@ -1043,7 +1043,8 @@
                                 "additionalProperties": false
                             },
                             "sshfp": {
-                                "description": "A Secure Shell fingerprint, used to verify the system’s authenticity",
+                                "description":
+                                        "A Secure Shell fingerprint, used to verify the system’s authenticity",
```
Comment (catenacyber) [inline etc/schema.json:1047]: This change does not look related to the commit :-p

**Review by catenacyber — CHANGES_REQUESTED** [review]
Looks good for the main thing, but some nit to fix

File: etc/schema.json (hunk 1 above)
Comment (jlucovsky) [reply etc/schema.json:1047]: This and other formatting changes are a result of clang-format.

Comment (catenacyber) [reply etc/schema.json:1047]: Needs its own commit anyways ;-p

**Comment by jlucovsky** [comment]
Continued in #10823
